src/tools/pkcs15-init.c

```diff
@@ -201,7 +202,8 @@ const struct option	options[] = {
 	{ "insecure",		no_argument, NULL,		OPT_INSECURE },
 	{ "use-default-transport-keys",
 				no_argument, NULL,		'T' },
-	{ "no-prompt",		no_argument, NULL,		OPT_NO_PROMPT },
+	{ "use-pinpad",		no_argument, NULL,		OPT_USE_PINPAD },
+	{ "no-prompt",		no_argument, NULL,		OPT_USE_PINPAD_DEPRECATED },
```

Comment:
Yes, also thought it was the better. Just didn't find any example of a silent option, in this code. Can you point me how to do it?